src/FSharpPlus/Data/NonEmptySeq.fs

```diff
@@ -15,11 +15,24 @@ type NonEmptySeq<'T> =
 type neseq<'t> = NonEmptySeq<'t>
 
 module NonEmptySeq =
-    let internal unsafeOfSeq (x: _ seq) =
+    /// <summary>Builds a non empty sequence from the given sequence.</summary>
+    /// <param name="seq">The input sequence.</param>
+    /// <returns>Non empty sequence containing the elements of the list.</returns>
+    /// <remarks>
+    ///   **This function does not check whether the sequence is actually non empty or not.**
+    /// 
+    ///   Use this function only if you are sure that the sequence is not empty and
+    ///   you don't want to evaluate the first element of the sequence which would cause a
+    ///   side effect.
+    ///  
+    ///   Otherwise, always use `ofSeq`. 
+    /// </remarks>
+    /// <seealso cref="ofSeq" />
+    let unsafeOfSeq (seq: _ seq) =
```

Comment:
Don't mean to be nitpicky, but I wonder if it's a good idea to use one of the most popular FSharp.Core function name as a parameter name.
FSharp.Core uses convention of `source` as a parameter name for seq to avoid shadowing.

Reply:
Is not that bad, I mean not that uncommon in F# code I think.

But I agree in that it’s better to use another name, I think FSharp.Core (implicit) convention is to call it `source`.

Reply:
That would be another PR. We also have this kind of naming in https://github.com/fsprojects/FSharpPlus/blob/master/src/FSharpPlus/Data/NonEmptyList.fs#L75 etc.